internal/api/observability/output_types_test.go

```diff
@@ -30,5 +34,55 @@ var _ = Describe("helpers for output types", func() {
 			}
 		})
 
+		It("should return an empty set of keys when authentication is not defined for an output", func() {
```

Comment:
#2533 alters the API to make this required which means in theory these tests likely would never be run in prod